bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/ButtonImpl.java

```diff
@@ -87,6 +88,14 @@ public String getButtonType() {
         return buttonType;
     }
 
+    @Override
+    public String getFieldType() {
```

Comment:
We could move the logic to compute field type in base class as a function, and re-use it everywhere else. Also, we can use the StringUtils.isBlank check for fieldTypeJcr to check empty strings as well

Reply:
Used StringUtils.IsNotEmpty to check for empty and null